graphql_schema.json

```diff
@@ -3732,7 +4053,7 @@
             },
             {
               "name": "pooledUserCommands",
-              "description": "Retrieve all the user commands submitted by the current daemon that are pending inclusion",
+              "description": "Retrieve all the scheduled user commands for a specified sender that the current daemon sees in their transaction pool. All scheduled commands are queried if no sender is specified",
```

Comment:
should we rename userCommand -> userTransactions as part of the renaming, or too much?

Reply:
lets leave that for later